waypoints/Alps-mountains.cup

```diff
@@ -0,0 +1,3884 @@
+name,code,country,lat,lon,elev,style,rwdir,rwlen,freq,desc
+"Achen-Kogel",Achengel,AT,4714.400N,01253.900E,2260.0m,7,,,,"Gipfel"
```

Comment:
Instead of Gipfel, please use the english word Mountain Top. A lot of xcsoar users are international.

Reply:
How about removing it altogether, because Style=7 already encodes is as mountain to. How2soar follows this logic.